faculty/clients/experiment.py

```diff
@@ -293,3 +333,34 @@ def list_runs(
         return self._get(
             endpoint, ListExperimentRunsResponseSchema(), params=query_params
         )
+
+    def log_run_data(
+        self, project_id, run_id, metrics=None, params=None, tags=None
+    ):
+        """Update the data of a run.
+
+        Parameters
+        ----------
+        project_id : uuid.UUID
+        run_id : uuid.UUID
+        metrics : List[experiments.ExperimentRunMetric], optional
```

Comment:
I'd just document these as `List[ExperimentRunMetric]` etc. since `experiments.ExperimentRunMetric` doesn't give you the full import path anyway - better to just accept that these types are defined in the same context as the client.

Reply:
👍